dashboard-notifications/src/test/java/uk/gov/hmcts/reform/dashboard/services/DashboardNotificationServiceTest.java

```diff
@@ -22,6 +30,24 @@ public class DashboardNotificationServiceTest {
     @InjectMocks
     private DashboardNotificationService dashboardNotificationService;
 
+    @Test
```

Comment:
Could we nest these @miguelMolina3691 So we know they belong together as getters?
You don't have to, I will approve without that, but see line 54.

Reply:
Sure, before merge it, there were no nested classes, and when solving the conflicts I didn't notice it. Sorted!